tools/render-test/render-d3d11.cpp

```diff
@@ -115,28 +109,28 @@ class D3D11Renderer : public Renderer, public ShaderCompiler
         /// Definition of the HLSL-to-bytecode compilation logic.
     static ID3DBlob* compileHLSLShader(char const* sourcePath, char const* source, char const* entryPointName, char const* dxProfileName);
         /// Capture a texture to a file
-    static HRESULT captureTextureToFile(ID3D11Device* dxDevice, ID3D11DeviceContext* dxContext, ID3D11Texture2D* dxTexture, char const* outputPath);
+    static HRESULT captureTextureToFile(ID3D11Device* device, ID3D11DeviceContext* context, ID3D11Texture2D* texture, char const* outputPath);
```

Comment:
A note on naming convention here: the premise behind my using, e.g., `dxTexture` etc. is to help distinguish the `Renderer`-level version of things (e.g., a `Texture2D` class that is part of the `Renderer` abstraction) from the D3D-level version of the same thing (e.g., `ID3D11Texture2D`). In functions that traffic in both layers, it helps to have a convention to distinguish them.

Anyway, I have no strong opinion on this rename, other than to reiterate that I'm wary of making too many pure "cleanup" changes on code that otherwise didn't need to be touched.

For some of the formatting stuff, the right answer is that we should probably set up `clang-format` to run on all of our code automatically, and enforce a uniform (if arbitrary) coding convention.